### ISSUE TITLE
[Improvement, Prio] Informing users about Tangem backups

### ISSUE THREAD

**Issue by WietseWind** [XRPL-Labs/XUMM-Issue-Tracker#376]
### Context

We are offering users XUMM Tangem cards for improved security. The improved security lies (but not limited to) mostly in the fact that users don't have their keys, don't have to write down their keys, can't make key entry errors and most importantly: can't compromise their keys by leaking their secret. They are not written down, not in a cloud account and users can't even enter them if a scammer asks them to.

When a Tangem card is lost or destroyed, account access has been lost forever. To mitigate this potential disastrous downside of key management where one can't access and back up the key, we offer the **Tangem Backup** xApp, using the XRPL `Regular Key` feature to assign a second account (second card) to the used account, so when lost/destroyed, account access can be restored using the r-address holding the funds, and the configured back up Tangem card (regular key) to sign.

The need to configure a back up card to prevent fund loss when a card is lost or destroyed must be very very clear to end users. This is why I propose to add a couple of flows/features to XUMM.

# 1. When importing a Tangem card

#### 1A. If the Tangem card does not contain a key pair yet (new account, first use) or if the imported account is not activated 

In this case we must inform (separate screen during import) the users that it's a good idea to configure a back up Tangem Card.

A disclaimer shows that if the user doesn't do this, their funds will be inaccessible forever if the card is lost or destroyed. 

At this stage the new account is not activated yet so we will just **inform the user that we will remind them every month until they set this up, and if they want to set it up before the reminder shows, they can open the "Tangem Backup" xApp**.

#### 1B. If the Tangem card already contains a key pair and the account is activated, and the account **does not contain** a Regular Key

This means the user used the card & account before, and as it is activated, users have the funds (fee can dip below reserve) to sign and submit a SetRegularKey transaction. Just like 1A: we inform the user (separate screen during import) that they best configure a back up account. They now have three options:

- 1. Skip, remind me later » A disclaimer shows that if the user doesn't do this, their funds will be inaccessible forever if the card is lost or destroyed. 
- 2. Set up » opens the "Tangem Backup" xApp.
- 3. Get another Tangem card » opens the "Order Tangem card" xApp

# 2. Monthly

For every Tangem based account, only once per month **per account**, and only if the account **does __not__ have a Regular Key**, XUMM must show a warning when the Home screen is selected on that account. 

The same flow as 1B applies here: we inform the user, and they can: **Skip** (Disclaimer), **Set up** or **Order another card**.

# 3. General improvements / to add
- [ ] Will/Patrick/...: something to write collaboratively, containing the QR straight to the Tangem Backup xApp and the QR to XUMM Support xApp. For now (until the things above and below exist) we must ship all Tangem card orders with a sheet of paper with instructions
- We must no longer sell the one (1) card package, the min. amount to order must be 2, so users have a good shot at configuring a back up account. We must then sell two different designs, so users can distinguish the cards they pick as primary and back up card.
- [ ] @thisistriss Same (non transparent) design, different layout/colours to have printed
- Tangem Back Up xApp: as users are routed here: better instructions on the xApp home page (what it can be used for, what the requirements are) and better instructions if the second card hasn't been imported yet (close, go to Settings - Accounts, import second card, no need to activate, then open Tangem Back Up xApp again)
- Inform screen: "Inform (separate screen during import) the users that it's a good idea to configure a back up Tangem Card."
- Disclamer screen: "A disclaimer shows that if the user doesn't do this, their funds will be inaccessible forever if the card is lost or destroyed. "
